### DISCUSSION TITLE
[14주차] 온라인 스터디

### DISCUSSION THREAD

**Discussion by ChaerinYu** [no-study-no-future/interview-study#94] [General]
~ 20:15 각자 PR리뷰
~ 21:00 발표
~ 21:10 회고 (좋았던 점, 건의사항, 느낀점 등)

**Comment by ChaerinYu** [comment]
### 발표
1. 유채린
* [Spring] filter와 interceptor (+AOP) #93
2. 김정윤
* [ETC] TDD란 #96
3. 이주형
* [OS] Inter Process Communication #98

**Comment by ChaerinYu** [comment]
### 회고 (a.k.a. 출석체크)
* 참석 확인용이므로 `감사합니다!` 만 작성해주셔도 좋아요 🍀
* 좋았던 점
* 건의사항
* 느낀점

**Reply by KJY97** [reply]
TDD 대신 MockMVC에 대해서 찾아봐야겠어요.. 감사합니다.

**Reply by ChaerinYu** [reply]
다들 고생하셨습니다~~~~(╹ڡ╹ )

**Reply by leeejuhyeong** [reply]
완성시켜서 다시 올릴게요...ㅎ..
